tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -513,110 +506,6 @@ describe(__filename, () => {
     expect(statsLink).toHaveLength(1);
   });
 
-  it('links to version history if add-on is extension', () => {
```

Comment:
Once you reintroduce the "history link" in the `AddonMoreInfo` component, these test cases will pass again. We should add them back :)

Reply:
Wow, added it back and it works!!